src/main/java/edu/ksu/canvas/model/sisimport/SisImportErrorsAttachment.java

```diff
@@ -0,0 +1,222 @@
+package edu.ksu.canvas.model.sisimport;
+import edu.ksu.canvas.annotation.CanvasField;
+import edu.ksu.canvas.annotation.CanvasObject;
+import edu.ksu.canvas.model.BaseCanvasModel;
+
+import java.io.Serializable;
+import java.time.Instant;
+
+/**
+ * Class to represent Canvas Sis Imports Errors Attachment
+ * See <a href="https://canvas.instructure.com/doc/api/sis_imports.html#SisImport">SisImport</a> documentation.
+ */
+ @CanvasObject(postKey = "errors_attachment")
```

Comment:
I'm not sure you need the `@CanvasObject` and `@CanvasField` annotations as this object shouldn't ever be POSTed back to Canvas. The only place they are used is in `edu.ksu.canvas.model.BaseCanvasModel` when building POST request parameters.

Reply:
Yeah, pure imitation, let me address this and re-test, thanks @buckett.